dll/cpl/timedate/internettime.c

```diff
@@ -93,12 +95,40 @@ SetNTPServer(HWND hwnd)
     UINT uSel;
     WCHAR szSel[4];
     LONG lRet;
+    WCHAR buffer[256];
 
     hList = GetDlgItem(hwnd,
                        IDC_SERVERLIST);
 
     uSel = (UINT)SendMessageW(hList, CB_GETCURSEL, 0, 0);
 
+    SendDlgItemMessage (hwnd, IDC_SERVERLIST, WM_GETTEXT, _countof(buffer), (LPARAM) buffer);
```

Comment:
Please use explicit unicode function `SendDlgItemMessageW` (and remove space before opening parenthesis). The extra space after the LPARAM cast is not needed.

Reply:
Corrected. Thanks.